Troubleshooting_Ipv6.md

```diff
@@ -0,0 +1,273 @@
+# IPv6 Disconnected Install Troubleshooting
+
+In this document we will analyze common scenarios of errors/failures which can cuase the cluster installation to fail. 
+
+## Common failures Basic checks before Installing
+
+Below are some precautions we should follow before even attempting the OCP4.3 Ipv6 deployments. 
+
+* [Ipv4-BM-IPI](https://github.com/openshift-kni/baremetal-deploy/blob/master/install-steps.md#preparing-the-provision-node-for-openshift-install) has some pre-requisites which will be applied fpr ipv6 deployments as well. 
```

Comment:
I think it's a bad idea to link out to a GitHub repo not associated with the product.

Reply:
:+1:

If there's something specific for baremetal IPI and IPv6, we do have docs in this repo for that (/docs/user/metal). Ideally baremetal-deploy docs that are generally relevant should be added there.